webapp/src/components/preview-movies/style/row/TopRow.tsx

```diff
@@ -16,6 +14,8 @@ type Props = {
 };
 
 function TopRow({ movieList, isDataLoading }: Props) {
+  const SLIDER_TIMING: number = 8000;
+
```

Comment:
constante hors du composant, sinon ce n'est pas une constante
```suggestion
const SLIDER_TIMING: number = 8000;
  
function TopRow({ movieList, isDataLoading }: Props) {
```